### ISSUE TITLE
Unintended white background

### ISSUE THREAD

**Issue by ImNowAPro** [DannyFeliz/vue-tweet#26]
Hi,
I would like to use your widget on a dark themed site, but the background is completely white.
![image](https://user-images.githubusercontent.com/45796033/214143191-2247bcb5-a469-4ef6-be1a-f6f6a3998e88.png)
 
Would be nice if you could fix it or give me instructions how to I can fix it myself.

**Comment by DannyFeliz** [comment]
Hi, 

This component does not add that white background; please try to change the background of the parent component of the vue-tweet component.

**Comment by ImNowAPro** [comment]
Hi,
I already did that and it looks like this:
![image](https://user-images.githubusercontent.com/45796033/214232908-28e6a482-dfda-49ac-b7a0-a76dc97456b6.png)
I think that the background-color is set by the component.

**Comment by DannyFeliz** [comment]
How can be that possible if in the component I don't even have a style tag 🙂 https://github.com/DannyFeliz/vue-tweet/blob/main/src/components/vue-tweet.vue

**Comment by ImNowAPro** [comment]
I don't know, maybe it comes from the tweet itself? Anyways it would nice if you could remove the background somehow, I think this would be helpful for everyone.

**Comment by Josepdal** [comment]
Hi @JakobJar have you found any solution? I am having the same problem, don't know if it is something related to Nuxt 3, will keep investigating...

**Comment by JakobJar** [comment]
> Hi @JakobJar have you found any solution? I am having the same problem, don't know if it is something related to Nuxt 3, will keep investigating...

No, I haven't... I have been using Vue 3 and switched to Nuxt 3 some time ago but the issue remained. I'm additionally using Ionic/Capacitor in my project so I thought it's an issue on that side but based on your comment that doesn't seem to be the case. Here is an issue on the Ionic forum I found in my research: https://forum.ionicframework.com/t/cannot-create-a-iframe-with-transparent-background-in-ionic-framework/224587
Would be happy if you could keep me up to date with your investgations!

**Comment by Josepdal** [comment]
> > Hi @JakobJar have you found any solution? I am having the same problem, don't know if it is something related to Nuxt 3, will keep investigating...
> 
> No, I haven't... I have been using Vue 3 and switched to Nuxt 3 some time ago but the issue remained. I'm additionally using Ionic/Capacitor in my project so I thought it's an issue on that side but based on your comment that doesn't seem to be the case. Here is an issue on the Ionic forum I found in my research: https://forum.ionicframework.com/t/cannot-create-a-iframe-with-transparent-background-in-ionic-framework/224587 Would be happy if you could keep me up to date with your investgations!

Hello, I have found a solution. For me, the iframe was inheriting the property color-scheme: dark and with that it was not working. To fix it I have added the following CSS code:

````
iframe
  color-scheme: auto
````

**Comment by DannyFeliz** [comment]
Closing since this is not something I control.
